examples/portworx/getting_started/main.tf

```diff
@@ -0,0 +1,127 @@
+  token                  = data.aws_eks_cluster_auth.this.token
+}
+
+provider "helm" {
+  kubernetes {
+    host                   = module.eks_blueprints.eks_cluster_endpoint
+    cluster_ca_certificate = base64decode(module.eks_blueprints.eks_cluster_certificate_authority_data)
+    token                  = data.aws_eks_cluster_auth.this.token
+  }
+}
+
+data "aws_eks_cluster_auth" "this" {
+  name = module.eks_blueprints.eks_cluster_id
+}
+
+data "aws_availability_zones" "available" {}
+
+
+#---------------------------------------------------------------
+# Supporting Resources
+#---------------------------------------------------------------
+
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 3.0"
+
+  name = local.name
+  cidr = local.vpc_cidr
+
+  azs             = local.azs
+  public_subnets  = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k)]
+  private_subnets = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k + 10)]
+
+  enable_nat_gateway   = true
+  single_nat_gateway   = true
+  enable_dns_hostnames = true
+
+  # Manage so we can name
+  manage_default_network_acl    = true
+  default_network_acl_tags      = { Name = "${local.name}-default" }
+  manage_default_route_table    = true
+  default_route_table_tags      = { Name = "${local.name}-default" }
+  manage_default_security_group = true
+  default_security_group_tags   = { Name = "${local.name}-default" }
+
+  public_subnet_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/elb"                      = 1
+  }
+
+  private_subnet_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/internal-elb"             = 1
+  }
+
+  tags = local.tags
+}
+
+
+#---------------------------------------------------------------
+# EKS Blueprints
+#---------------------------------------------------------------
+
+module "eks_blueprints" {
+  source = "../../.."
+
+  cluster_name    = local.cluster_name
+  cluster_version = "1.22"
+
+  vpc_id             = module.vpc.vpc_id
+  private_subnet_ids = module.vpc.private_subnets
+
+  managed_node_groups = {
+    eksblueprint_nodegroup_med_1 = {
+      node_group_name = "eksblueprint_nodegroup_med_1"
+      instance_types  = ["t2.medium"]
+      min_size        = 3
+      desired_size    = 3
+      max_size        = 3
+      subnet_ids      = module.vpc.private_subnets
+    }
+  }
+  tags = local.tags
+}
+
+
+
+module "eks_blueprints_kubernetes_addons" {
+
+  source               = "../../../modules/kubernetes-addons"
+  eks_cluster_id       = module.eks_blueprints.eks_cluster_id
+  eks_cluster_endpoint = module.eks_blueprints.eks_cluster_endpoint
+  eks_oidc_provider    = module.eks_blueprints.oidc_provider
+  eks_cluster_version  = module.eks_blueprints.eks_cluster_version
+
+
+
+  enable_portworx = true
+  portworx_chart_values = {
+    awsAccessKeyId     = var.aws_access_key_id
```

Comment:
Can we replace this example with the policy example? Ideally, policy should be created automatically or inlined for the node group role.

Reply:
What do you mean when you say should be created automatically

Reply:
You can create the policy with terraform or just add inline policies to the nodegroup role.

Reply:
If you look at this [commit ](https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/955/commits/e3230aef59039525487e526572a877b803c14bbf) which is also a part of this PR, I have removed the AWS credentials flow and all of its documentation as suggested and only kept the Policy workflow. Somehow you are still seeing the outdated version for the files. Can you check it again on your end